lib/plugins/wallet/monerod/monerod.js

```diff
@@ -5,14 +5,16 @@ const jsonRpc = require('../../common/json-rpc')
 
 const { COINS, utils } = require('lamassu-coins')
 
-const blockchainUtils = require('../../../coin-utils')
 const BN = require('../../../bn')
 const E = require('../../../error')
 const { logger } = require('../../../blockchain/common')
+const options = require('../../../options')
+
+const blockchainDir = options.blockchainDir
```

Comment:
In the file `lib/blockchain/do-volume.js` you read the blockchain dir from the environment. Is that a different variable, or should this be read from the environment as well?